docs/dev-tools/chopsticks/index.md

```diff
@@ -0,0 +1,395 @@
+|      `wasm-override`       |             Path of the WASM to use as the parachain runtime, instead of an endpoint's runtime              |
+|            `db`            |               Path to the name of the file that stores or will store the parachain's database               |
+|          `config`          |                                       Path or URL of the config file                                        |
+|           `port`           |                                      The port to expose an endpoint on                                      |
+|     `build-block-mode`     |                       How blocks should be built in the fork: batch, manual, instant                        |
+|      `import-storage`      |              A pre-defined JSON/YAML storage file path to override in the parachain's storage               |
+| `allow-unresolved-imports` |              Whether to allow WASM unresolved imports when using a WASM to build the parachain              |
+|           `html`           |                           Include to generate storage diff preview between blocks                           |
+|   `mock-signature-host`    | Mock signature host so that any signature starts with `0xdeadbeef` and filled by `0xcd` is considered valid |
+
+### Using a Configuration File
+
+The Chopsticks source repository includes a collection of [YAML](https://yaml.org/){target=_blank} files that can be used to set up various Substrate chains locally. You can download these configuration files from the [repository's `configs` folder](https://github.com/AcalaNetwork/chopsticks/tree/master/configs){target=_blank}.
+
+An example of a configuration file for Polkadot is as follows:
+
+```yaml
+endpoint:
+  - wss://rpc.ibp.network/polkadot
+  - wss://polkadot-rpc.dwellir.com
+mock-signature-host: true
+block: ${env.POLKADOT_BLOCK_NUMBER}
+db: ./db.sqlite
+runtime-log-level: 5
+
+import-storage:
+  System:
+    Account:
+      -
+        -
+          - 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+        - providers: 1
+          data:
+            free: '10000000000000000000'
+  ParasDisputes:
+    $removePrefix: ['disputes'] # those can makes block building super slow
+```
+
+To run Chopsticks using a configuration file, utilize the `--config` flag. You can use a raw GitHub URL, a path to a local file, or simply the chain's name. For example, the following commands all use Polkadot's configuration in the same way:
+
+=== "GitHub URL"
+
+    ```bash
+    npx @acala-network/chopsticks@latest \
+    --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/polkadot.yml
+    ```
+
+=== "Local File Path"
+
+    ```bash
+    npx @acala-network/chopsticks@latest --config=configs/polkadot.yml
+    ```
+
+=== "Chain Name"
+
+    ```bash
+    npx @acala-network/chopsticks@latest --config=polkadot
+    ```
+
```

Comment:
Just being nit-picking again, I'd suggest adding a small note that specifies the following:

"In case that you need to interact with a specific version of chopstick, you should replace @latest on commands above by @VERSION". Or something like that...

WDYT?

Reply:
I removed the @latest from all the commands. All the examples are performed with a local chopsticks installation in a project, so the version should be managed within the `package.json`